main.cpp

```diff
@@ -43,6 +17,9 @@ int main(int argc, char **argv)
         .default_value(std::size_t{1})
         .scan<'u', std::size_t>();
```

Comment:
There is something not right with `scan<'u'....`.

For options `-l` and `-n` I receive very strange help:
" -l, --l        length of generated sequences **[nargs=0..1]** [default: 8]"
What `nargs` suggests? Is that mean that proper value range is from 0-1?
That's not appropriate for both options.